doc/mkdocs/docs/api/basic_json/find.md

```diff
@@ -2,10 +2,10 @@
 
 ```cpp
 template<typename KeyT>
-iterator find(KeyT&& key);
+iterator find(KeyT && key);
```

Comment:
Unless you intend find() to create the item if it doesn't exist, you'd want to take const& here, instead of &&.